HTML/HTML.sublime-syntax

```diff
@@ -98,20 +98,23 @@ contexts:
         1: punctuation.definition.tag.begin.html
         2: entity.name.tag.script.html
       push:
-        - match: (?i)(</)(script)(>)
+        - match: (?i)(-->)?(</)(script)(>)
           captures:
             0: meta.tag.script.end.html
-            1: punctuation.definition.tag.begin.html
-            2: entity.name.tag.script.html
-            3: punctuation.definition.tag.end.html
+            1: comment.block.html punctuation.definition.comment.html
+            2: punctuation.definition.tag.begin.html
+            3: entity.name.tag.script.html
+            4: punctuation.definition.tag.end.html
           pop: true
-        - match: '>'
-          scope: meta.tag.script.begin.html punctuation.definition.tag.end.html
+        - match: '(>)(<!--)?'
+          captures:
+            1: meta.tag.script.begin.html punctuation.definition.tag.end.html
+            2: comment.block.html punctuation.definition.comment.html
           push:
             - meta_content_scope: source.js.embedded.html
             - include: 'scope:source.js'
           with_prototype:
-             - match: (?i)(?=</script)
+             - match: (?i)(?=(-->)?</script)
```

Comment:
Adding a `\s*` after `(-->)?` would allow whitespace if the `-->` is on the same line as the script closing tag, but if there is a line break, the `-->` would not be highlighted, correct? I ask because I have a similar problem with Graphviz.